projects/vdk-control-cli/.gitlab-ci.yml

```diff
@@ -84,7 +84,8 @@ vdk-control-cli-release:
     - pip install -U pip setuptools wheel twine
     - python setup.py sdist --formats=gztar
     - twine upload --repository-url "$PIP_REPO_UPLOAD_URL" -u "$PIP_REPO_UPLOAD_USER_NAME" -p "$PIP_REPO_UPLOAD_USER_PASSWORD" dist/*tar.gz --verbose
-  only:
-    refs:
-      - main
-    changes: *vdk_control_cli_changes_locations
+  rules:
```

Comment:
have we covered all the release/publish stages of all artifacts at that point?

Reply:
there is no stage called publish, there is `publish_artifacts`, `release` and  `release_image`. and yeah they are all covered.

Reply:
Yes, I went looking for it, and found vdk-core gitlab ci yaml file for example is also not patched.

I managed to do it by fetching all the latest changes in the project, then click find in projects/ dir while narrowing the file types to .y*ml, and looking for each occurrence of "publish" and "release" words.

Attaching a screenshot of how I did the check:
<img width="1792" alt="Screenshot 2023-01-26 at 13 57 36" src="https://user-images.githubusercontent.com/11415233/214830379-5243a410-d2ba-4e6b-ae2f-9b3229577b08.png">

Reply:
Oh sorry! thanks!
Should be all fixed now